### PR TITLE
Feature/aside

### PR DESCRIPTION
**Pull request by Singh-Ish**
Info 
Its the aside alert block 
- added the block 
- added the story and the test 

Jira Ticket No: https://itsjira.carleton.ca/browse/WSDEV-861

### CODE REVIEW

File: src/components/AlertAside/AlertAside.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+        if (type === 'error') {
+            it(`${type.toUpperCase()}: Should render close button`, () => {
+                cy.get(button).should('exist')
+                cy.get(`${button} .u-icon`).should('exist')
+                cy.get(`${button} .c-icon`).should('exist')
+            })
+        }
+    })
+})
```
Comment (marceloglacial) [inline src/components/AlertAside/AlertAside.spec.tsx:55]: Good job. 
One more thing. 
Could you, please, add tests to verify when the user click on the close button??
Thank you

File: src/components/AlertAside/AlertAside.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+export default {
+    component: AlertAside,
+    title: 'Components/AlertAside',
+} as Meta
+
+const Template: Story = (args: any) => <>
+<Layout type='ma'>
+        <Main>
+            <h2>Multicol MA (Main + Aside)</h2>
```
Comment (marceloglacial) [inline src/components/AlertAside/AlertAside.stories.tsx:18]: Awesome!

Could you, please change the title and the content for the title and the two first paragraphs of the RDS documentation?
Thank you 

https://ravendesignsystem.github.io/rds/dev/blocks/aside/alertaside/

**Review by marceloglacial — CHANGES_REQUESTED** [review]

File: src/components/AlertAside/AlertAside.stories.tsx (hunk 2 above)
Comment (Singh-Ish) [reply src/components/AlertAside/AlertAside.stories.tsx:18]: Thank you marcelo I have done he necessary changes to it

File: src/components/AlertAside/AlertAside.spec.tsx (hunk 1 above)
Comment (Singh-Ish) [reply src/components/AlertAside/AlertAside.spec.tsx:66]: Hi Marcelo, 
I have added the alert on click test for the alert aside block . kindly check . Thanks !!

**Review by marceloglacial — APPROVED** [review]
Good Job :)

**Pull request merged**